client/my-sites/site-settings/form-writing.jsx

```diff
@@ -119,13 +123,15 @@ class SiteSettingsFormWriting extends Component {
 					/>
 				) }
 
-				{ siteIsJetpack && config.isEnabled( 'press-this' ) && (
+				{ ( renderPostByVoice || renderPostByEmailAndPressThis ) && (
 					<PublishingTools
 						isAtomic={ isAtomic }
 						onSubmitForm={ handleSubmitForm }
 						isSavingSettings={ isSavingSettings }
 						isRequestingSettings={ isRequestingSettings }
 						fields={ fields }
+						renderPostByVoice={ renderPostByVoice }
+						renderPostByEmailAndPressThis={ renderPostByEmailAndPressThis }
```

Comment:
I'm not a big fan of passing this kind of props, we should be able to move that conditional logic in the `PublishingTools` component itself (and render nothing if needed). I also don't understand why `SiteSettingsFormWriting` as well as`PublishingTools` have some logic for showing the `Press This` section.

Reply:
>I also don't understand why SiteSettingsFormWriting as well asPublishingTools have some logic for showing the Press This section.

Oh, yeah, I haven't noticed it. And it's the reason why you see different section here (https://github.com/Automattic/wp-calypso/pull/90489#pullrequestreview-2055720429).
I was testing somehow and didn't see the second component, hm

Reply:
@stephanethomas JFI I have refactored the duplication (https://github.com/Automattic/wp-calypso/pull/90489/commits/875f35118129ba8a003aa47be0bf931cdd922010), and as a result - separated "Publishing tools" sections are fixed (now it's always one section)

Reply:
>I'm not a big fan of passing this kind of props, we should be able to move that conditional logic in the PublishingTools component itself (and render nothing if needed). 

Also addressed this one ^^^